packages/node_modules/@webex/internal-plugin-encryption/src/kms.js

```diff
@@ -661,6 +663,22 @@ const KMS = WebexPlugin.extend({
     context.ephemeralKey = originalContext.ephemeralKey;
 
     return context;
+  },
+
+  updateKmsKey(activity) {
```

Comment:
This method looks as though it could have an integration test. Please add one to validate these changes work.

Reply:
Will add integration test for this function, once resolve below mentioned locally testing problem.